CMakeLists.txt

```diff
@@ -58,9 +58,9 @@ option(WITH_SUPERCOP   "Build Ed25519 using the ref10 implementation from SUPERC
 option(WITH_TESTS      "Build unit tests" OFF)
 option(WITH_FUZZ_TESTS "Build fuzz tests" OFF)
 option(WITH_UPNP       "Include support for UPnP client" OFF)
+option(WITH_COTIRE     "Enable cotire (compile time reducer) - precompiled header and single compilation unit builds" ${MSVC})
 
 
-set(CMAKE_VERBOSE_MAKEFILE true)
```

Comment:
No. This is useful for debugging on all our buildbot machines - and I literally just put this in this past week. I'm not going to ask Dan to maintain `VERBOSE=1`.

Reply:
From IRC
```
<btolfa> anonimal about https://github.com/monero-project/kovri/pull/867 - I don't like by default set(CMAKE_VERBOSE_MAKEFILE true), how about detecting buildbot environment from Makefile and use VERBOSE=1 only for buildbot's builds?
<anonimal> 1) that sounds like unnecessary work 2) we won't need to say "VERBOSE=1" every time someone submits a build log to debug
<anonimal> What's the big deal? It's more helpful than not.
<btolfa> It makes output a bit noisy. Ok, I will make it enabled by default, but overridable from cmake options - like option(CMAKE_VERBOSE_MAKEFILE "" ON) instead of set(CMAKE_VERBOSE_MAKEFILE TRUE)
<anonimal> Ok, an option() is fine. Please reference the PR # so we know why.
```